proto/topodata.proto

```diff
@@ -396,6 +402,20 @@ message ThrottlerConfig {
   map <string, double> metric_thresholds = 7;
 }
 
+// KeyspaceVtorcConfig stores configuration for Vtorc.
+message KeyspaceVtorcConfig {
+  // DisableEmergencyReparent reflects if EmergencyReparentShard
+  // can be used in Vtorc recoveries.
+  bool disable_emergency_reparent = 1;
+}
+
+// ShardVtorcConfig stores configuration for Vtorc.
+message ShardVtorcConfig {
```

Comment:
`ShardVtorcConfig` is currrently == `KeyspaceVtorcConfig` but I plan to add at least 1 new keyspace-only field to `KeyspaceVtorcConfig` in a future PR